### PR TITLE
Feature/desktop improvements

### PR DESCRIPTION
**Pull request by duffleit**
Added a layout which is appearable on the desktop. 

image

### CODE REVIEW

File: src/layouts/AppLayout/AppLayout.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import classes from "./AppLayout.scss";
+import "../../styles/core.scss";
+
+export const AppLayout = ({children}) => (
```
Comment (thomaseizinger) [inline src/layouts/AppLayout/AppLayout.js]: I think this named export is redundant.

File: src/layouts/AppLayout/AppLayout.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
+.layout{
+  @media #{$sm-min}  {
```
Comment (thomaseizinger) [inline src/layouts/AppLayout/AppLayout.scss]: What do you think about renaming these breakpoints to something more meaningful? Like $mobile?

Comment (duffleit) [reply src/layouts/AppLayout/AppLayout.scss]: good point

File: src/layouts/AppLayout/AppLayout.js (hunk 1 above)
Comment (duffleit) [reply src/layouts/AppLayout/AppLayout.js]: you are right

**Pull request merged**